[2025-05-03T12:34:07.402+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-03T12:34:07.462+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews.model_data manual__2025-05-03T12:33:00.938514+00:00 [queued]>
[2025-05-03T12:34:07.495+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews.model_data manual__2025-05-03T12:33:00.938514+00:00 [queued]>
[2025-05-03T12:34:07.514+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-03T12:34:07.547+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): model_data> on 2025-05-03 12:33:00.938514+00:00
[2025-05-03T12:34:07.568+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2515) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-03T12:34:07.573+0000] {standard_task_runner.py:72} INFO - Started process 2530 to run task
[2025-05-03T12:34:07.576+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews', 'model_data', 'manual__2025-05-03T12:33:00.938514+00:00', '--job-id', '265', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmpmb6_keql']
[2025-05-03T12:34:07.578+0000] {standard_task_runner.py:105} INFO - Job 265: Subtask model_data
[2025-05-03T12:34:07.661+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews.model_data manual__2025-05-03T12:33:00.938514+00:00 [running]> on host 8280c28b7f0f
[2025-05-03T12:34:07.853+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews' AIRFLOW_CTX_TASK_ID='model_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-03T12:33:00.938514+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-03T12:33:00.938514+00:00'
[2025-05-03T12:34:07.858+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-03T12:34:07.860+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-03T12:34:07.860+0000] {logging_mixin.py:190} INFO - Current task name:model_data state:running start_date:2025-05-03 12:34:07.464644+00:00
[2025-05-03T12:34:07.861+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews and current dag run status:running
[2025-05-03T12:34:07.862+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-03T12:34:07.881+0000] {base.py:84} INFO - Retrieving connection 'postgres_defaut'
[2025-05-03T12:34:08.378+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-03T12:34:08.398+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-03T12:34:08.399+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews, task_id=model_data, run_id=manual__2025-05-03T12:33:00.938514+00:00, execution_date=20250503T123300, start_date=20250503T123407, end_date=20250503T123408
[2025-05-03T12:34:08.449+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-03T12:34:08.450+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-03T12:34:08.451+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews queued_at:2025-05-03 12:33:00.965062+00:00
[2025-05-03T12:34:08.452+0000] {logging_mixin.py:190} INFO - Task hostname:8280c28b7f0f operator:PythonOperator
[2025-05-03T12:34:08.476+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-03T12:34:08.546+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-03T12:34:08.567+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
